lib/entitlement_info_wrapper.dart

```diff
@@ -38,13 +38,13 @@ class EntitlementInfo {
   /// The date an unsubscribe was detected. Can be `null`.
   /// @note: Entitlement may still be active even if user has unsubscribed.
   /// Check the `isActive` property.
-  final String unsubscribeDetectedAt;
+  final String? unsubscribeDetectedAt;
 
   /// The date a billing issue was detected. Can be `nil` if there is no
```

Comment:
we should probably update the end of this line to "can be `null` if there is no"